src/data/screenshots.json

```diff
@@ -30,250 +30,250 @@
-                    { "filename": "/assets/screenshots/2-21/en/android/check_in_host_view_create_qr_code_category_selection", "alt": "Check In image 2 of 8" },
-                    { "filename": "/assets/screenshots/2-21/en/android/check_in_host_view_create_qr_code_attributes", "alt": "Check In image 3 of 8" },
-                    { "filename": "/assets/screenshots/2-21/en/android/check_in_host_view_display_qr_code", "alt": "Check In image 4 of 8" },
-                    { "filename": "/assets/screenshots/2-21/en/android/check_in_host_view_qr_code_print_version", "alt": "Check In image 5 of 8" },
-                    { "filename": "/assets/screenshots/2-21/en/android/check_in_attendee_view_my_check_ins", "alt": "Check In image 6 of 8" },
-                    { "filename": "/assets/screenshots/2-21/en/android/check_in_attendee_view_check_in_detail_view", "alt": "Check In image 7 of 8" },
-                    { "filename": "/assets/screenshots/2-21/en/android/check_in_attendee_view_contact_journal", "alt": "Check In image 8 of 8" }
+                    { "filename": "/assets/screenshots/2-22/en/android/check_in_host_view_my_qr_codes", "alt": "Check In image 1 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/android/check_in_host_view_create_qr_code_category_selection", "alt": "Check In image 2 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/android/check_in_host_view_create_qr_code_attributes", "alt": "Check In image 3 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/android/check_in_host_view_display_qr_code", "alt": "Check In image 4 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/android/check_in_host_view_qr_code_print_version", "alt": "Check In image 5 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/android/check_in_attendee_view_my_check_ins", "alt": "Check In image 6 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/android/check_in_attendee_view_check_in_detail_view", "alt": "Check In image 7 of 8" },
+                    { "filename": "/assets/screenshots/2-22/en/android/check_in_attendee_view_contact_journal", "alt": "Check In image 8 of 8" }
                 ]
             },
             {
                 "title": "Statistics",
                 "anchor": "android_stats",
                 "description": "In this section you will find screenshots of the statistics. If you scroll down on the \"Status\" page, you will see \"Add local 7-day incidence\". Now swipe to the right to read the current numbers. The numbers available are the 7-day incidence, COVID-19 Patients in Intensive Care Units, confirmed new infections, warnings by app users, 7-day R-value, people who have received at least one vaccine dose, people who have basic immunization, people who have received a booster vaccination and the vaccine doeses administered.",
```

Comment:
doeses => doses

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/2785/commits/3e69001bd580758463a610bf9e1dcf7148f2a277, thanks!